spec/active_record_spec.cr

```diff
@@ -36,8 +36,23 @@ module ActiveRecord
     end
   end
 
+  class DependentsIn < NullAdapter::Query
+    def call(params, fields)
+      puts params
+      puts fields
+      return false unless fields.has_key?("number_of_dependents") &&
+                          params.has_key?("1")
+      actual = fields["number_of_dependents"] as Int
+      low = params["1"] as Int
+      high = params["2"] as Int
+      actual > low && actual < high
+    end
+  end
+
   NullAdapter.register_query("(number_of_dependents > :1) AND (number_of_dependents < :2)",
     LessAndMoreDependents.new)
+
+  NullAdapter.register_query("number_of_dependents IN ((:1,:2))", DependentsIn.new)
```

Comment:
I've found it :smile: The extra whitespace was necessary in query `(number_of_dependents) IN ((:1, :2))`